platform/viewer/cypress/support/commands.js

```diff
@@ -410,3 +409,33 @@ function canvasToImage(selectorOrEl) {
   canvas.parentElement.appendChild(image);
   canvas.style = 'display: none';
 }
+
+//Initialize aliases for User Preferences modal
+Cypress.Commands.add('initPreferencesModalAliases', () => {
+  initPreferencesModalAliases();
+});
+
+Cypress.Commands.add('openPreferences', () => {
+  // Open User Preferences modal
+  cy.get('[data-cy="options-menu"]')
+    .scrollIntoView()
+    .click()
+    .then(() => {
+      cy.get('[data-cy="about-item-menu"]')
+        .last()
+        .click();
+    });
+});
+
+Cypress.Commands.add('resetHotkeysPreferences', () => {
```

Comment:
This opens user preferences but does not close them. Can we rename it to `resetHotkeyBindings` and make sure it closes the `UserPreferences` modal?

Reply:
Or if you prefer "preferences": `resetUserHoktkeyPreferences`

Reply:
On the current flow, once user clicks on "Reset to Defaults" button, the modal is automatically closed.